source/features/copy-file.tsx

```diff
@@ -1,26 +1,44 @@
 import React from 'dom-chef';
 import select from 'select-dom';
 import copyToClipboard from 'copy-text-to-clipboard';
+import delegate from 'delegate-it';
 import features from '../libs/features';
 
 function handleClick({currentTarget: button}: React.MouseEvent<HTMLButtonElement>): void {
 	const file = button.closest('.Box');
-
 	const content = select.all('.blob-code-inner', file!)
 		.map(blob => blob.innerText) // Must be `.innerText`
 		.map(line => line === '\n' ? '' : line)
 		.join('\n');
-
 	copyToClipboard(content);
 }
 
-function init(): void {
-	// This selector skips binaries + markdowns with code
-	for (const code of select.all('.blob-wrapper > .highlight:not(.rgh-copy-file)')) {
-		code.classList.add('rgh-copy-file');
-		code
-			.closest('.Box')! // Closest common container
-			.querySelector('[data-hotkey="b"]')! // Easily-found `Blame` button
+async function init(): Promise<void> {
+	// This won't show button on markdown and binary files until the code is loaded
+	if (select.exists('.blob.instapaper_body')) {
```

Comment:
Now that I think about it, we don't need this `if` at all.

It's easier this way:

```js
renderButton();
delegate('.rgh-md-source', 'rgh:view-markdown-source', renderButton);
delegate('.rgh-md-source', 'rgh:view-markdown-rendered', () => {
	const button = select('.rgh-copy-file');
	if (button){
		button.remove()
	}
});

Reply:
that should render the button on markdown files when they're rendered.

Reply:
as apposed to being in source.

Reply:
or I imagine you count on rendered firing and removing the button?

Reply:
that doesn't work

Reply:
changing import order didn't work either

Reply:
Indeed, the selector is no longer `.blob-wrapper > .highlight` so `renderButton` was called even on rendered markdown. I updated it